answers.yaml.example

```diff
@@ -0,0 +1,7 @@
+dotfiles_repo: git@github.com:myuser/dotfiles.git
+fedora_packages:
+  my-fave-text-editor
+  my-alternate-web-browser
+fedora_packages_to_remove:
+  something-i-dislike
+  that-other-thing-i-dislike
```

Comment:
Ya, I was thinking in my head of doing ```-e @answers.yaml``` but thats silly, it should definitely be in the playbook.  As for the clone and include.. i love it and was wondering if you could do that.